homeassistant/components/switchbot_cloud/binary_sensor.py

```diff
@@ -43,6 +46,32 @@ class SwitchBotCloudBinarySensorEntityDescription(BinarySensorEntityDescription)
     on_value="opened",
 )
 
+MOVE_DETECTED_DESCRIPTION = SwitchBotCloudBinarySensorEntityDescription(
+    key="moveDetected",
+    device_class=BinarySensorDeviceClass.MOTION,
+    value_fn=lambda data: data.get("moveDetected") is True
+    or data.get("detectionState", "") == "DETECTED",
```

Comment:
```suggestion
    value_fn=lambda data: data.get("moveDetected") is True
    or data.get("detectionState") == "DETECTED",
```
Also please wrap multiline lambdas in parenthesis